tests/python/gpu/test_operator_gpu.py

```diff
@@ -310,6 +310,38 @@ def check_fast_lars(w_dtype, g_dtype, shapes, ctx, tol1, tol2):
             ref_new_lrs[i] = lrs[i]
     assert_almost_equal(ref_new_lrs.asnumpy(), mx_new_lrs.asnumpy(), atol=tol2, rtol=tol2)
 
+def check_multi_sum_sq(dtype, shapes, ctx, tol1, tol2):
+    values_arr = [np.random.rand(*shape).astype(dtype) * 10. for shape in shapes]
+
+    mx_vals = _make_ndarrays(values_arr, ctx=ctx)
+    sum_sq = mx.nd.multi_sum_sq(*mx_vals, num_arrays=len(shapes))
+    sum_sq2 = mx.nd.multi_sum_sq(*mx_vals, num_arrays=len(shapes))
+    # checks that operator is deterministic
+    assert_almost_equal(sum_sq.asnumpy(), sum_sq2.asnumpy(), atol=1e-9, rtol=1e-9)
```

Comment:
instead of using assert_almost_equal, shall we use `np.array_equal` for `sum_sq.asnumpy(), sum_sq2.asnumpy()`? It checks if two arrays are exactly the same

Reply:
yes that is better, thanks